src/components/CreatePool/index.tsx

```diff
@@ -32,108 +38,109 @@ function CreatePool({ history }) {
   const token0: Token = useToken(token0Address)
   const token1: Token = useToken(token1Address)
 
-  const [step, setStep] = useState<number>(1)
+  const [step, setStep] = useState<string>(STEP.SELECT_TOKENS)
 
   const pair = usePair(token0, token1)
-  const pairExists = // used to detect new exchange
-    pair && JSBI.notEqual(pair.reserve0.raw, JSBI.BigInt(0)) && JSBI.notEqual(pair.reserve1.raw, JSBI.BigInt(0))
 
+  // if both tokens selected but pair doesnt exist, enable button to create pair
   useEffect(() => {
-    if (token0Address && token1Address && pair && !pairExists) {
-      setStep(2)
+    if (token0Address && token1Address && !pair) {
```

Comment:
should this be `pair === null`? if pair is undefined it's just loading right?

Reply:
yep good catch - should only enable if its specifically null